.github/workflows/run-tests.yml

```diff
@@ -271,6 +271,30 @@ jobs:
           echo "$TEST_RESULTS_WALLET" | base64 -d > wallet.json
         working-directory: neofs-testcases
 
+      - name: Define expire
```

Comment:
i think `expiry` or `expiration` is a better choice

Reply:
`expiration`.

Reply:
Fixed